addons/settings/fnc_init.sqf

```diff
@@ -89,6 +89,7 @@ switch (toUpper _settingType) do {
     };
     case "EDITBOX": {
         _defaultValue = _valueInfo param [0, "", [""]]; // don't use params - we want these variables to be private to the main scope
+        _settingData pushBack (_valueInfo param [1, false]);
```

Comment:
Just to be safe, maybe add type checking `param [1, false, [false]]`

Reply:
I already have `isEqualTo true` check in the place where I read this one.